learn/getting_started/quick_start.md

```diff
@@ -69,35 +69,36 @@ Below is an example to showcase how to add documents to an index called `movies`
 
 [API references](/reference/api/documents.md)
 
-### Checking updates
+### Checking task status
 
-Most actions in MeiliSearch are [asynchronous](/learn/advanced/asynchronous_updates.md), including the document addition process.
+Most actions in MeiliSearch are [asynchronous](/learn/advanced/asynchronous_operations.md), including the document addition process.
 
-Asynchronous actions return a JSON object that contains only an `updateId` attribute. This is a **successful response**, indicating that the operation has been taken into account, but may not have been executed yet.
+Asynchronous actions return the following response in the stated order:
 
-You can check the status of the operation via the `updateId` and the [get update status route](/reference/api/updates.md). Checking the update status of an operation is never mandatory, but can prove useful in tracing the origin of errors or unexpected behavior.
+| Field      | Type    | Description                              |
+|------------|---------|---------------------------------         |
+| uid        | integer | Unique sequential identifier             |
+| indexUid   | string  | Unique index identifier                  |
+| status     | string  | Status of the task. Value is `enqueued`  |
+| enqueuedAt | string  | Represents the date and time in the ISO 8601 format when the task has been `enqueued`                                                        |
```

Comment:
I don’t know where else to add it. I added it here because they will see this response so might as well explain what it does. The only other place is asynchronous operations. I can try fitting it there